src/SDKs/Resource/Resource.Tests/InMemoryTests/DeploymentTests.InMemory.cs

```diff
@@ -135,7 +135,7 @@ public void DeploymentTestsCreateValidateMessage()
             Assert.True(result.Properties.Outputs.ToString().Contains("\"type\": \"string\""));
         }
 
-        [Fact]
+        [Fact(Skip = "Failing Deployment Test")]
```

Comment:
You don't have to skip this, if you revert the SerializeObject call in DeploymentOperations (see my comment there)

Reply:
Done. Thanks